### ISSUE TITLE
Other Organizations

### ISSUE THREAD

**Issue by sillyisfun231** [Xeyler/joinlater#4]
Given that this works for Utah State University, would tweaking the details make it work for other organizations?
Would it work for other organizations as long as the enrollment URL, CRT file, organization ID, profile ID, and Renew Window Days are known? I do not have the organization's name.

Thanks!

**Comment by brighamcampbell** [comment]
Yes, the latest changes should, in theory, make it possible to authenticate with many more SecureW2 deployments. I would expect it to work with any organization that uses a single sign-on webpage to authenticate and uses certificate-based WPA2 Enterprise to secure the connection.

I have only tested joinlater at USU, but I’m interested in making it work with any SecureW2 deployment.
